pandas/tseries/tests/test_timeseries.py

```diff
@@ -4461,6 +4461,23 @@ def test_second(self):
         self.assertIsInstance(r2, Float64Index)
         tm.assert_index_equal(r1, r2)
 
+class TestDaysInMonth(tm.TestCase):
+    def test_day_not_in_month_coerce_true_NaT(self):
+        self.assertTrue(isnull(to_datetime('2015-02-29', coerce=True)))
```

Comment:
Added comments and blank lines. Should I open new issues for these test cases that do not pass?
```
self.assertTrue(isnull(to_datetime('2015-02-29', format="%Y-%m-%d", coerce=True)))
self.assertTrue(isnull(to_datetime('2015-04-31', format="%Y-%m-%d", coerce=True)))
```